Gemfile

```diff
@@ -23,6 +23,8 @@ gem "sprockets-rails"
 gem "turbo-rails"
 gem "stimulus-rails"
 
+gem "config"
```

Comment:
Looks like active development for over 10 years, with a good number of downloads, and minimal dependencies
<img width="1092" height="926" alt="image" src="https://github.com/user-attachments/assets/d67d9d5f-aa74-4bd1-b916-6cec80327309" />